browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -50,11 +52,80 @@ test.describe('program creation', () => {
     await adminProgramImage.expectProgramImagePage()
   })
 
-  test('create program with disabled visibility condition feature enabled', async ({
```

Comment:
Removed `'disabled visibility condition feature enabled'` from the test name since this feature was rolled out, so we no longer manually enable it on the test. Similarly for the screenshot name
(that way is test name and screenshot name are less confusing)